.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "deps/JetStream"]
```

Comment:
Does it mean the default is head? In automation test or integration test, we would need the tag or commit id for the project.

Reply:
the default is this hash: 
![image](https://github.com/google/jetstream-pytorch/assets/1719482/04c2e0bc-3ab9-4988-9eb0-390ede0c58f9)

To update this hash one ch's into the directory and run `git pull ..` etc.